src/tribler-common/tribler_common/utilities.py

```diff
@@ -27,6 +29,43 @@ def uri_to_path(uri):
 
 
 fts_query_re = re.compile(r'\w+', re.UNICODE)
+tags_re = re.compile(r'^\s*(?:\[[^\s\[\]]{3,50}\]\s*)+')
```

Comment:
**issue:** We discussed search string format again, and I still think that the format `"[tag-1][tag-2] FTS search string"` may be problematic. It is very common for torrent titles to start with some text in square brackets. So, if a user copies and pastes such a torrent name into the search field, it can get unexpected query results, as the first part of the title will be erroneously interpreted as a tag.

**suggestion:** The alternative possible format is `"#tag-1 #tag-2 FTS search string"`. Torrent names almost never start with a hash sign. It is about 2000 times less frequent than torrent names which start with square brackets. Also, it is a bit easier to use when manually entering the query string - no need to add a closing square bracket at the end of the tag.

Reply:
Implemented with one detail. Tags could be placed anywhere within a query string.